benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -31,6 +31,21 @@
 
 struct Spec
 {
+  Spec(std::string const &backends, int const n_values, int const n_queries,
+       int const n_neighbors, int const sort_predicates_int,
+       int const buffer_size, PointCloudType const source_point_cloud_type,
+       PointCloudType const target_point_cloud_type)
+      : backends(backends)
+      , n_values(n_values)
+      , n_queries(n_queries)
+      , n_neighbors(n_neighbors)
+      , sort_predicates_int(sort_predicates_int)
+      , buffer_size(buffer_size)
+      , source_point_cloud_type(source_point_cloud_type)
+      , target_point_cloud_type(target_point_cloud_type)
+  {
+  }
+
```

Comment:
You can't use aggregate initialization if you provide a constructor. We can have a `create_spec_from_string_function` instead of the constructor taking an `std::string`, though, if you prefer.